python/oneflow/test/modules/test_deconv2d.py

```diff
@@ -937,6 +937,16 @@ def test_deconv2d_group_with_random_data(test_case):
         y = m(x)
         return y
 
+    @profile(torch.nn.functional.conv_transpose2d)
+    def profile_conv_transpose2d(test_case):
+        inputs = torch.ones(16, 3, 128, 128)
+        weights_64c = torch.ones(3, 64, 4, 4)
```

Comment:
```suggestion
        weights_4x4_64c = torch.ones(3, 64, 4, 4)
```

另外 input 的 channel 是 3 是不是也不符合常见情况

此外可以再加一个包含 output_padding 的测试

Reply:
已修改，增加上采样3倍的操作